src/LocalField/neq.jl

```diff
@@ -796,6 +798,7 @@ function local_fundamental_class_serre(mKL::LocalFieldMor)
   rE, mE = residue_field(E)
   rL, mL = residue_field(L)
   rK, mK = residue_field(K)
+  # how is this supposed to work?
   mrKL = hom(rK, rL, mL(mKL(preimage(mK, gen(rK)))))
```

Comment:
Without promotion magic, this only works if `base_field(rK) === base_field(rL)`, but this is not the case here.